examples/chain.js

```diff
@@ -1,9 +1,9 @@
-var Nemo = require("../");
-var config = {
-  "driver": {
-    "browser": "phantomjs"
-  },
-  'data': {
+const Nemo = require('../');
+const config = {
+  'driver': {
+    'browser': 'phantomjs'
+  }
+  ,'data': {
```

Comment:
@grawk , @skratchdot , actually '--fix' parameter auto-fix it. Although this looks small thing but let me observe this behavior ... and will update soon on this.